<p>This rule detects hardcoded secrets</p>
<h2>Noncompliant Code Example</h2>
<pre>
TO DO
</pre>
<h2>Compliant Solution</h2>
<pre>
Use a keystore to store all secrets and reference them by symbolic IDs in your code.
</pre>
